01/index.html

```diff
@@ -1,13 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <meta name="author" content="Mateusz Bogolubow">
-    <title>devmentor.pl - JS Tools - #01</title>
-</head>
-<body>
-    <p>GitHub też jest fajny!</p>
-</body>
-</html>
\ No newline at end of file
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<meta name="author" content="Mateusz Bogolubow" />
+		<title>devmentor.pl - JS Tools - #01</title>
+	</head>
+	<body>
+		<h1>GIT jest fajny! :)</h1>
+		<p>GitHub też jest fajny!</p>
+	</body>
+</html>
```

Comment:
To jest komentarz do konkretniej linii kodu ;) 
Ty już wiesz jak to wygląda na GH :P